src/client/pps/pps.proto

```diff
@@ -342,6 +360,15 @@ message RestartDatumRequest {
   repeated string data_filters = 2;
 }
 
+message InspectDatumRequest {
+  string datum_id = 1 [(gogoproto.customname) = "DatumID"];
+  string job_id = 2 [(gogoproto.customname) = "JobID"];
+}
```

Comment:
The other requests in this file use wrapper structs for identifiers, i.e. rather than `string job_id = 2;` they use `Job job = 2;` would you mind doing it that way for consistency. You'll need to create a `Datum` message too.